public/assets/js/scrirpts.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function(){
+    $('img.plusone').hide();
+    $('img.minustwo').hide();
+    $('img.minusthree').hide();
+    
+
+    $('.benefits-slider').slick({
+        speed: 500,
+        infinite: true,
+        slidesToShow: 3,
+        slidesToScroll: 1,
+        autoplay: true,
+        autoplaySpeed: 2000,
+        centerPadding: '3rem',
+        prevArrow: '<img src="./assets/img/btn-prev.png" alt="previous-arrow" class="slider-prev arrow">',
+        nextArrow: '<img src="./assets/img/btn-next.png" alt="next-arrow" class="slider-next arrow">',
+    });
+
+    $(".one").click(function() {
```

Comment:
Instead of manually creating the accordion, I think it is more preferable to use bootstrap's default accordion but to edit the CSS to match the design. The plus image is shown when the collapse is open (via CSS) and changed to minus when closed. https://getbootstrap.com/docs/4.3/components/collapse/